platform-operator/controllers/verrazzano/component/common/vmi.go

```
@@ -78,6 +79,42 @@ func DeleteVMI(ctx spi.ComponentContext) error {
 	return nil
 }
 
+// IsLegacyOS returns true if the OS that is running is managed by VMO
+func IsLegacyOS(ctx spi.ComponentContext) bool {
```

Return `(bool, error)`, and let the caller decide whether it wants to ignore errors.